Description: libsystemd-login-dev to libsystemd-dev transition in Debian stretch
Author: Michael Biebl <biebl@debian.org>

diff --git a/configure.ac b/configure.ac
index c518280..5378b13 100644
--- a/configure.ac
+++ b/configure.ac
@@ -940,7 +940,7 @@ AC_ARG_WITH(systemd,
             [with_systemd=$withval], [with_systemd=auto])
 
 PKG_CHECK_MODULES(SYSTEMD,
-                  [libsystemd-login],
+                  [libsystemd],
                   [have_systemd=yes], [have_systemd=no])
 
 if test "x$with_systemd" = "xauto" ; then
